parts/linux/cloud-init/artifacts/manifest.json

```diff
@@ -4,11 +4,10 @@
         "downloadLocation": "/opt/containerd/downloads",
         "downloadURL": "https://moby.blob.core.windows.net/moby/moby-containerd/${CONTAINERD_VERSION}+azure/bionic/linux_${CPU_ARCH}/moby-containerd_${CONTAINERD_VERSION}+azure-${CONTAINERD_PATCH_VERSION}_${CPU_ARCH}.deb",
         "versions": [
-            "1.4.9-3",
```

Comment:
dropped since contains CVE and it's unused in latest CSE/VHD, only here for old clusters which node image upgrade without CSE changes.